int32 femurLength
int32 tibiaLength
float64 roughness
float64 hardness
float64 cot
